controllers/accounts.js

```diff
@@ -2,42 +2,170 @@
 * controllers/accounts.js
 */
 const controller = module.exports = {}
-const uuidv4 = require('uuid/v4'),
-      sgMail = require('@sendgrid/mail');
+const connection = require('../app/dbconn'),
+      uuidv4 = require('uuid/v4'),
+      sgMail = require('@sendgrid/mail'),
+      crypto = require('crypto');
       
-const models = require('../models').sequelize.models
+const models = require('../models').sequelize.models;
 const Account = models.account;
+const Contact = models.Contact;
+
 const auth = require('./auth')
 controller.createAccount = (req,res) => {
-    const account = req.body,
-    uuid = uuidv4();
-    account.id = uuid;
+    
+    const account = req.body
+    const contact = {} // create contact
+    contact.id = uuidv4();
+    account.contact = contact.id;
+    contact.first_name = account.firstName;
+    contact.last_name = account.lastName;
+    contact.email = account.email;
+    Contact.create(contact);
+    account.id = uuidv4();
     account.password = '';
     account.isActivated = 0;
     account.createdBy = req.session.user.id;
     
     Account.create(account).then((account)=>{
+        
         sgMail.setApiKey(process.env.SENDGRID_API_KEY);
         const msg = {
           to: `${account.email}`,
           from: `noreply@agendamanager.nl`,
           subject: `Registration at agendamanager.nl`,
           text: `Someone has invited you to join Agenda Manager. Visit agendamanager.nl/verify and paste the following code: ${account.id}`,
-          html: `${req.session.user.firstName} ${req.session.user.lastName} has invited you to join Agenda Manager. Visit <a href="${process.env.REF_URL}verify?uuid=${account.id}">agendamanager.nl/verify</a> and paste the following code: <br><strong>${account.id}</strong>`,
+          html: `<div><img src="https://dev.agendamanager.nl/img/logo_sm_300.png" style="margin:auto;" /></div>${req.session.user.firstName} ${req.session.user.lastName} has invited you to join Agenda Manager. Visit <a href="${process.env.REF_HTTP_PROTOCOL}://${process.env.REF_URL}verify?uuid=${account.id}">agendamanager.nl/verify</a> and paste the following code: <br><strong>${account.id}</strong>`,
```

Comment:
Is dit plaatje altijd aanwezig??? (als de server niet draait dan werkt het wel???

Reply:
Is zeker geen definitieve oplossing, al zou de server we moeten draaien als de e-mail gelezen wordt (en het plaatje wordt getoond) ... anders werkt de bevestigingslink ook niet